task_hive/.github/workflows/sonarcloud.yml

```
@@ -1,40 +1,48 @@
-name: SonarCloud Code Analysis
+name: SonarQube Analysis
 
 on:
-  push:
-    branches:
-      - main
   pull_request:
-    branches:
-      - main
+    branches: [ main ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)